forms-flow-bpm/forms-flow-bpm-integration/src/main/resources/application.properties

```diff
@@ -0,0 +1,10 @@
+spring.application.name=Formsflow Integration
+
+
+# E-mail configurations
+spring.mail.host=smtp.gmail.com
+spring.mail.port=587
+spring.mail.username=sneha.suresh@aot-technologies.com
+spring.mail.password=mscvohwtkkbqggzm
```

Comment:
please remove the password and username.
Also can we move this to yaml as we follow yaml in bpm for properies.
And shouldn't this be on the main package than the sub-module ?